main/php.h

```diff
@@ -231,17 +231,7 @@ typedef unsigned int socklen_t;
 #include "zend_hash.h"
 #include "zend_alloc.h"
 #include "zend_stack.h"
-
-#if STDC_HEADERS
-# include <string.h>
-#else
-# ifndef HAVE_MEMCPY
-#  define memcpy(d, s, n)	bcopy((s), (d), (n))
-# endif
-# ifndef HAVE_MEMMOVE
```

Comment:
Wanted to suggest the same here, but apparently the bundled PCRE uses that one.

Reply:
The memmove checks can be removed when the pcre starts avoiding those checks, yes...